packages/p2p/src/stores/my-ads-store.js

```diff
@@ -22,24 +22,29 @@ export default class MyAdsStore extends BaseStore {
     current_method = { key: null, is_deleted: false };
     delete_error_message = '';
     edit_ad_form_error = '';
+    error_code = '';
```

Comment:
Most of the changes in the observables, actions and computed are just me rearranging it alphabetically